cmd/root.go

```diff
@@ -73,7 +74,16 @@ func initConfig() {
 
 	viper.AutomaticEnv()
 
-	if err := viper.ReadInConfig(); err == nil {
-		fmt.Fprintln(os.Stderr, "Using config file:", viper.ConfigFileUsed())
+	err := viper.ReadInConfig()
+
+	var notFound *viper.ConfigFileNotFoundError
+	switch {
+	case err != nil && !errors.As(err, notFound):
+		cobra.CheckErr(err)
+	case err != nil && errors.As(err, notFound):
```

Comment:
The message for this case is a new message on stderr.  I worry it will give the impression that a configuration is required.  How about we don't print anything when a config file is not present, just like now?